samples/matter/lock/sample.yaml

```diff
@@ -22,24 +25,32 @@ tests:
       - nrf7002dk/nrf5340/cpuapp/nrf7001
     platform_allow: nrf52840dk/nrf52840 nrf5340dk/nrf5340/cpuapp nrf7002dk/nrf5340/cpuapp
       nrf7002dk/nrf5340/cpuapp/nrf7001
+    tags: sysbuild
   sample.matter.lock.no_dfu.no_fd:
+    sysbuild: true
     build_only: true
     integration_platforms:
       - nrf21540dk/nrf52840
     platform_allow: >
       nrf21540dk/nrf52840
+    tags: sysbuild
   sample.matter.lock.thread_wifi_switched.smp_dfu:
+    sysbuild: true
     build_only: true
-    extra_args: SHIELD=nrf7002ek multiprotocol_rpmsg_SHIELD=nrf7002ek_coex
-      CONF_FILE=prj_thread_wifi_switched.conf CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_STORE=y
+    extra_args: lock_SHIELD=nrf7002ek multiprotocol_rpmsg_SHIELD=nrf7002ek_coex
```

Comment:
This configuration seems not to work. There is a problem with `MCUBOOT_UPDATEABLE_IMAGES` because I'm getting an error: `warning: default value 2 on MCUBOOT_UPDATEABLE_IMAGES ... clamped to 3 due to being outside the active range ([3, 3])`.
Does it mean that currently, sysbuild doesn't support 3 mcuboot images?